server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -63,6 +64,7 @@
 public abstract class FieldMapper extends Mapper {
     private static final Logger logger = LogManager.getLogger(FieldMapper.class);
 
+    public static final FeatureFlag DOC_VALUES_SPARSE_INDEX = new FeatureFlag("doc_values_sparse_index");
```

Comment:
I added the feature flag here so that we can later use this also for `DataStreamTimestampFieldMapper`